package.json

```diff
@@ -36,7 +36,15 @@
       "rangeStrategy": "bump",
       "npm": {
         "commitMessageTopic": "{{prettyDepType}} {{depName}}"
-      }
+      },
+      "packageRules": [
+        {
+          "groupName": "nuxt core",
+          "groupSlug": "nuxt",
+          "matchPackageNames": ["nuxt", "nuxt3", "nuxt-edge"],
+          "matchPackagePatterns": ["^@nuxt/"]
```

Comment:
This can be tricky because there is also packages like `@nuxt/content` and `@nuxt/image` there. Have you experienced what is renovate behavior for those cases? (I think explicitly listing framework deps would be better idea)